src/components/BackgroundIcon/BackgroundIcon.stories.tsx

```diff
@@ -5,20 +5,68 @@ import {withDesign} from "storybook-addon-designs";
 import {faSearch, faStar} from "@fortawesome/pro-solid-svg-icons";
 import {faStar as faStarLight} from "@fortawesome/pro-light-svg-icons";
 
-// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+const summary =
+  "##Summary\nThe Background icon component may contain various icons to visualize or to give information as an image.\n\n";
+const context =
+  "##Context\nThe Background component was added to create a visual of the information it is trying to portray.\n\n";
+const figmaLink =
+  "##Figma Link\nhttps://www.figma.com/file/yJ8kQwyMxuTZudD90MxgOU/%F0%9F%93%90-Components-and-Patterns?type=design&node-id=103-320&mode=design&t=QFaUGXlsf04huhew-0";
+const markdown = summary + context + figmaLink;
```

Comment:
Suggestion; can we add another section to the template that is something like; "Suggested use" that gives tips for usage, and perhaps some examples of where it's used.

Reply:
An example of where this would be useful is the Avatar component.

Apparently the "circle" variant should be used for users, whilst the "square" variant should be used for other entities. This should be captured in the documentation.